tools/topology/sof-cml-rt5682.m4

```diff
@@ -45,7 +45,7 @@ PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
 
 # Low Latency capture pipeline 2 on PCM 0 using max 2 channels of s24le.
 # 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-volume-capture.m4,
+PIPELINE_PCM_ADD(sof/pipe-`CPROC'-capture.m4,
```

Comment:
So in cases like this we should use the endpoint specific macro. In this case if could be HSMICPROC.

The passing of filter blob is the challenge. Since we here know that pipeline number is 2. We could set macro PIPE2_FILT1 from HSMICFILT1.

Reply:
I am actually asking if quoting `CPROC' here will still allow it to be expanded. Maybe use `'CPROC`' to still allow expansion?

Reply:
@paulstelian97 current code is expanded well. Otherwise it will throw error that it could find the file sof/pipe-CPROC-capture.m4.